packages/neo-one-server-plugin-project/src/utils/loadProjectConfig.ts

```diff
@@ -1,17 +1,24 @@
+import { CodegenFramework } from '@neo-one/smart-contract-codegen';
 import convict from 'convict';
 import cosmiconfig from 'cosmiconfig';
-import * as path from 'path';
+import fs from 'fs-extra';
+import path from 'path';
 import { ProjectConfig, projectConfigSchema } from '../types';
 
-const validateConfig = (rootDir: string, configIn: cosmiconfig.Config): ProjectConfig => {
+const validateConfig = async (rootDir: string, configIn: cosmiconfig.Config): Promise<ProjectConfig> => {
   const config = convict<ProjectConfig>(projectConfigSchema);
   config.load(configIn);
   config.validate({ allowed: 'warn' });
 
   const validatedConfig = config.getProperties();
+  const framework = await getProjectFramework(rootDir);
```

Comment:
This will always overwrite - this should only be called (and added) if the config is not present.